app/controllers/CheckYourAnswersController.scala

```diff
@@ -17,16 +17,21 @@
 package controllers
 
 import controllers.actions.{DataRequiredAction, DataRetrievalAction, IdentifierAction}
+import models.{NormalMode, UserAnswers}
```

Comment:
Some imports are not required if you remove the unnecessary construction arguments.

````
import models.{NormalMode, UserAnswers}
import org.slf4j.LoggerFactory
import pages.ContactNamePage
````